boards/arm/96b_aerocore2/96b_aerocore2.dts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <st/f4/stm32f427vi.dtsi>
+
+/ {
+	model = "96Boards Gumstix AeroCore 2";
+	compatible = "gumstix,aerocore2", "st,stm32f427";
+
+	chosen {
+		zephyr,console = &uart7;
+		zephyr,shell-uart = &uart7;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,ccm = &ccm0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		yellow_led_1: led_1 {
+			gpios = <&gpioe 10 GPIO_INT_ACTIVE_HIGH>;
```

Comment:
Please rebase on latest maser and update vs new GPIO API.

Reply:
done.